superbench/benchmarks/micro_benchmarks/rocm_gemm_flops_performance.py

```diff
@@ -0,0 +1,167 @@
+
+        self._bin_name = 'rocblas-bench'
+        self._support_precisions = ['FP64', 'TF32_TC', 'FP16_TC', 'BF16_TC', 'INT8_TC']
+        self.__precision_and_kernel_map = {
+            'FP64': '-r f64_r -f gemm',
+            'TF32_TC': '-r f32_r -f gemm_ex --compute_type f32_r',
+            'FP16_TC': '-r f16_r -f gemm_ex --compute_type f32_r',
+            'BF16_TC': '-r bf16_r -f gemm_ex --compute_type f32_r',
+            'INT8_TC': '--a_type i8_r --b_type i8_r --c_type i32_r --d_type i32_r -f gemm_ex --compute_type i32_r'
+        }
+
+    def add_parser_arguments(self):
+        """Add the specified arguments."""
+        super().add_parser_arguments()
+
+        self._parser.add_argument(
+            '--transposeA',
+            type=str,
+            default='N',
+            help='Transpose type of Matrix A, N = no transpose, T = transpose, C = conjugate transpose',
+        )
+        self._parser.add_argument(
+            '--transposeB',
+            type=str,
+            default='T',
+            help='Transpose type of Matrix B, N = no transpose, T = transpose, C = conjugate transpose',
+        )
+        self._parser.add_argument(
+            '--lda',
+            type=int,
+            default=8384,
+            required=False,
+            help='Leading dimension of matrix A.',
+        )
+        self._parser.add_argument(
+            '--ldb',
+            type=int,
+            default=8384,
+            required=False,
+            help='Leading dimension of matrix B.',
+        )
+        self._parser.add_argument(
+            '--ldc',
+            type=int,
+            default=8384,
+            required=False,
+            help='Leading dimension of matrix C.',
+        )
+        self._parser.add_argument(
+            '--ldd',
+            type=int,
+            default=8384,
+            required=False,
+            help='Leading dimension of matrix D.',
+        )
+        self._parser.add_argument(
+            '--alpha',
+            type=int,
+            default=1,
+            required=False,
+            help='Specifies the scalar alpha.',
+        )
+        self._parser.add_argument(
+            '--beta',
+            type=int,
+            default=0,
+            required=False,
+            help='Specifies the scalar beta.',
+        )
+
+    def _preprocess(self):
+        """Preprocess/preparation operations before the benchmarking.
+
+        Return:
+            True if _preprocess() succeed.
+        """
+        if not super()._preprocess():
+            return False
+
+        transpose_types = ['N', 'T', 'C']
+        if self._args.transposeA not in transpose_types or self._args.transposeB not in transpose_types:
+            self._result.set_return_code(ReturnCode.INVALID_ARGUMENT)
+            logger.warning(
+                'Unsupported transpose type - benchmark: {}, expected: {}.'.format(self._name, transpose_types)
+            )
+            return False
```

Comment:
using choices=['N', 'T', 'C'] in argparse?

Reply:
added, thanks for your suggestion